charts/argo-cd/templates/argocd-application-controller/statefulset.yaml

```diff
@@ -52,6 +52,9 @@ spec:
       {{- with .Values.controller.priorityClassName | default .Values.global.priorityClassName }}
       priorityClassName: {{ . }}
       {{- end }}
+      {{- if .Values.controller.terminationGracePeriodSeconds }}
+      terminationGracePeriodSeconds: {{ .Values.controller.terminationGracePeriodSeconds }}
+      {{- end }}
```

Comment:
I wonder if we need `if` condition since the type of `terminationGracePeriodSeconds ` is int. 👀 
If users wanted to turn off it, then they are supposed to set `0` (or just use the default value).